src/frontend/js/modules/box-overlay/box-overlay-manager.service.js

```diff
@@ -7,8 +7,8 @@ require('./box-overlay.constants.js');
 
   angular.module('esn.box-overlay').service('boxOverlayManager', boxOverlayManager);
 
-  function boxOverlayManager($rootScope, $window, $compile, $http, $templateCache, notificationFactory, ESN_BOX_OVERLAY_MAX_WINDOWS) {
-    var boxTemplateUrl = '/views/modules/box-overlay/box-overlay.html';
+  function boxOverlayManager($rootScope, $window, $compile, $q, notificationFactory, ESN_BOX_OVERLAY_MAX_WINDOWS) {
+    const boxTemplate = require('./box-overlay.pug');
     var boxes = [];
```

Comment:
const here, please

Reply:
There is a const here. What do you expect ?

Reply:
I mean the line below `var boxes = [];`

Reply:
I did not touch this line. I will not rewrite the entire file, this is not a refactor task here :-)

Reply:
ok :-)